### ISSUE TITLE
Expand the utility

### ISSUE THREAD

**Issue by amazingandyyy** [amazingshellyyy/covid19-api#1]
TL;DR;

Suggest to rename the repo name and change data path under docs to follow ISO standard.

## 2. Change path under docs

The structure can be **formatted** to be 
```
https://amazingshellyyy.com/covid19-data/<country-code>-<subdivisions-code>/<data-type>.json
```

### < country-code>

- follow [ISO 3166 Standard Country Codes](https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes)
- Example: United States is `US`, Taiwan is `TW`, United Kindom is `GB`

### < subdivisions-code>

- follow [
ISO 3166 — Codes for the representation of names of countries and their subdivisions](https://www.iso.org/obp/ui/#iso:code:3166:US)
- Example: California of United States is `US-CA`, New York of United States  is `US-NY`

### < data-type>

- can be timeseries or current, can be expanded

**Comment by amazingandyyy** [comment]
Here is the example, to get time-series data of each county of California of [US](https://www.iso.org/obp/ui/#iso:code:3166:US), the endpoint is 
```
https://amazingshellyyy.com/covid19-data/US-CA/counties/timeseries.json
```

Another example, to get current data of each city of Taipei of [Taiwan](https://www.iso.org/obp/ui/#iso:code:3166:TW), the endpoint is 
```
https://amazingshellyyy.com/covid19-data/TW-TPE/counties/current.json
```
